src/routes/safe/components/Balances/Coins/index.tsx

```diff
@@ -30,6 +31,17 @@ import { makeStyles } from '@material-ui/core/styles'
 import { styles } from './styles'
 import { currentCurrencySelector } from 'src/logic/currencyValues/store/selectors'
 
+const StyledButton = styled(Button)`
+  &&.MuiButton-root {
+    margin: 4px 12px 4px 0px;
+    padding: 0 12px;
+    min-width: auto;
+  }
+  svg {
+    margin: 0 6px 0 0;
+  }
+`
```

Comment:
if this is a common style. We can move it to a common location (`src/components`) and use it from there. WDYT?

it's also used in https://github.com/gnosis/safe-react/pull/2072/files#diff-59be34dd0325096fd42827ed294425603976629820911a9d1eb7c9af227e73ddR45-R54